railties/lib/rails/application.rb

```diff
@@ -111,8 +111,7 @@ def find_root(from)
       public :new
     end
 
-    attr_accessor :assets, :sandbox
-    alias_method :sandbox?, :sandbox
```

Comment:
And `Rails.application.sandbox?` could be added if it'll ease the transition to new logic.